src/BullOak.Repositories.EventStore.Test.Integration/Specification/ReadEventsSpecs.feature

```diff
@@ -0,0 +1,66 @@
+﻿Feature: ReadEventsSpecs
+	In order to support stream migration
+	As a user of this library
+	I want to be able to read all events from a particular stream
+
+Scenario: Read all events from a stream
+   Given a new stream
+	 And 3 new events
+     And I try to save the new events in the stream through their interface
+    When I read all the events back from the stream
+    Then the load process should succeed
+     And I should see all the events
```

Comment:
"the load process should succeed" looks redundant given we are checking for "I should see all the events"

Tests below do not check for "the load process should succeed" and I think this is how it should be.